docs/index.rst

```diff
@@ -27,6 +27,7 @@ Contents:
   Contributing to SIMP <contributors_guide/index>
   Security Concept of Operations <security_conop/index>
   Security Control Mapping <security_mapping/index>
+  Vulnerability Supplement <vulnerability_supplement/index>
```

Comment:
A supplement is something that enhances, we are not enhancing vulnerabilities, but I don't care enough to have this changed unless others take issue.